internal/integration/script_test.go

```diff
@@ -135,6 +137,11 @@ func (t *liteTest) setupScript(env *testscript.Env) error {
 	// environment as tests run in parallel.
 	env.Values[keyDB] = db
 	env.Values[keyDrv] = drv
+	cliPath, err := compileCLI(t.T)
```

Comment:
Why not storing in the temporary directory (of testscript)?

Reply:
dont want to compile it for every test case